.gitignore

```diff
@@ -1 +1,5 @@
 node_modules
+<<<<<<< HEAD
```

Comment:
Is there any reason why you have this here?

Reply:
It's an oversight, I had issues with rebasing the initial conflicts. It's been fixed